api.ts

```diff
@@ -322,6 +323,10 @@ export interface SyncTableDef<
 
   /** See {@link SyncTableOptions.description} */
   description?: string;
+  /** See {@link SyncTableOptions.contentCategory}
+   * @hidden
+   */
+  contentCategory?: ContentCategory;
```

Comment:
Is there a logic behind putting this on the sync table vs in the schema? It would seem that this is describing the data, and not the table that syncs it. I don't feel too strongly though.

Reply:
Yeah, good point. We are not always consistent, e.g. identityName is very much about the schema but syntactically people preferred it on the table since it was required. I'm happy to see it moved to the schema, it does feel like it's annotating the data in a similar way to other semantic properties.

One pressure test though, would we imagine that we also categorize other building blocks like action formulas, in order to indicate that e.g. they are using for sending emails? If so, that would be labeling the formula, would it feel inconsistent if sync tables were labeled on the schema?

Reply:
Maybe? As I understand it, this category is a targeted feature for helping Brain understand Pack output, and not part of some larger plan to apply taxonomy to building blocks, so that hypothetical feels unlikely. I'm fine with either approach, but I think it's good to have a rule of thumb on when to use one vs the other.